third_party/python/riotwatcher/riotwatcher.py

```diff
@@ -244,6 +244,10 @@ def request_available(self):
 
 
 class RiotWatcher:
+
+    # Used for testing purpose, disabling https make all routes testable.
+    enable_https = True
```

Comment:
Used for testing purpose < i.e. do not modify this for production code
disabling https make all routes testable. < creating an HTTPS server for tests is not a funny thing to do :)
enable_https = True < required by the Riot API

Should I detail a little bit more the comment?